docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -239,6 +239,8 @@ The markers layer allows you to display data points as different marker shapes s
 
 - **Size** configures the size of the markers. The default is `Fixed size`, which makes all marker sizes the same regardless of the data; however, there is also an option to size the markers based on data corresponding to a selected field. `Min` and `Max` marker sizes have to be set such that the markers can scale within this range.
 - **Symbol** allows you to choose the symbol, icon, or graphic to aid in providing additional visual context to your data. Choose from assets that are included with Grafana such as simple symbols or the Unicon library. You can also specify a URL containing an image asset. The image must be a scalable vector graphic (SVG).
+- **Symbol Vertical Align** configures the vertical alignment of the symbol relative to the data point. Note: This option is impacted by the symbol's rotation angle.
```

Comment:
```suggestion
- **Symbol Vertical Align** configures the vertical alignment of the symbol relative to the data point. Note: The symbol's rotation angle is applied first about the data point, then alignment displacement is applied.
```

Reply:
@drew08t - "about the data point" - I'm not sure what's meant by that. Would "for" or "on" work instead of "about"? Same question for line 243.